tap_universal_file/tap.py

```diff
@@ -16,6 +17,8 @@
 
     import click
 
+SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
```

Comment:
This won't work for users in production who would point to universal-file as a git repo / pypi package